platform/src/common/components/Map/components/MapNodes.js

```diff
@@ -103,6 +115,16 @@ export const getAQIcon = (pollutant, value) => {
   }
 };
 
+export const getAQIMessage = (pollutant, value) => {
+  if (!markerDetails.hasOwnProperty(pollutant)) {
+    throw new Error(`Invalid pollutant: ${pollutant}`);
+  }
+
+  const aqiCategory = getAQICategory(pollutant, value);
+
+  return messages[aqiCategory?.icon] || '';
```

Comment:
Hi @Codebmk , what is your view of the optional chaining operator (`?.`)?

Appears that it is supported in modern JavaScript environments, but it's not available in older versions of JavaScript so we might want to rethink things as we go about its usage --- it is supported in Node.js versions 14 and above, and in most modern browsers. However, it's not available in older JavaScript environments.

Reply:
We are using node 14, so this shouldn't be an issue

Reply:
Cool, thanks @Codebmk